components/transform/transform.app.mjs

```diff
@@ -1,11 +1,71 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "transform",
-  propDefinitions: {},
+  propDefinitions: {
+    formId: {
+      type: "string",
+      label: "Form ID",
+      description: "The form",
```

Comment:
Could you add a more explanatory description?

Reply:
Oops, actually I'm going to remove the description